### PR TITLE
Docs-72: Initial setup of the docs directory (port website/docs content to restructured text/sphinx)

### PR DESCRIPTION
**Pull request by kay-kim**
# Port of docs to restructured text/sphinx

## Rendered View

Can see rendered view: https://kay.inrupt.net/public/docs-staging/solid-client-js/index.html

### 1st commit 
- To ensure minimal impact, created a new docs/ folder, separate from the website folder.
- The source files are located in docs/source.
- The javascript code snippets are located in docs/source/examples.
- tsconfig.eslint.json has been updated to include docs/source/examples.
- This pull is just the initial laying the structure in github.  More editing, etc. will be done.  But wanted to get the general structure/framework in place. 

### 2nd commit
- Name change to solid-client-js

### 3rd commit 
- Inclusion of the feedback widget on the pages

### 4th commit
- Port of the manage ACL
### 5th commit
- Tweak note on trusted applications to a topic
### 6th commit
- Update configuration and files to use .rst for restructured text instead of .txt

### 7th commit
Updated per feedback.  Unfortunately, haven’t had a chance to try anything out. 

-  Install all 3 libraries.
-  Called out Vocabulary in the Read/Write Data (as well as a little bit in install) 
-  Copied the solid-client-authn from the pull request’s readme into a Prereq for Read/Write Data
-  LitDataset → SolidDataset, including the API names in code and links.
-  Updated Glossary.
-  Updated links in Read/Write Data to go to glossary.
-  I think the vocab-common-rdf is what is being meant by generated vocab artifacts?  In any case, changed to import { FOAF } from "@inrupt/vocab-common-rdf"; 
-  Updated blurb in Issues & Help to point to Service Desk for product.  Per discussion, will keep for now the separate Docs JIRA.  
-  Added links to schema.org, node.js, and commonjs.

### 8th commit
- Trivial - forgot to delete old link to github issues which was replaced by ServiceDesk link.

### 9th commit

- Prep for going live. Set indexing on and include matomo tracking script.

### 10th commit
- Add short cut icon to the pages

## General Details

- The  Guide> Motivations page was not ported.  We mention interoperability on the landing page and the other regarding documentation/testing/non-goals don't seem to belong in the docs.

- The  Guide>  Core Concepts page was not ported as same information found in the Working with Data Page.

- The Guide > Next Steps page:  1) An API link in the left-hand Nav;  2) Connect with Community and Contribute content has been ported to Issues & Help. 3) Didn't port Common Tasks section.

- Tutorials > Working with Data page  ported as Read/Write Data.

- Tutorials > Getting Started page was not ported since wasn't that different from Read/Write Data.

- Tutorials > Working with Files page  ported as  Read/Write Files

- Tutorials > Managing Access ported.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+rst_epilog = '\n'.join([
+    replacement_string,
+])
+
+pygments_style = 'sphinx'
+
+# -- Using .txt instead of .rst for restructured text -----------------------
+source_suffix = {
+    '.txt': 'restructuredtext',
```
Comment (Vinnl) [inline docs/conf.py:39]: I was playing around with this branch a bit and noticed this. Would you mind too much if we just used `.rst` for ReST? Then I can set up my editor to do automatic syntax highlighting and such for ReST files :)

Comment (kay-kim) [reply docs/conf.py:39]: that's fine.

Comment (kay-kim) [reply docs/conf.py]: Updated to use .rst

**Comment by Vinnl** [comment]
I'm going to close this PR; I've incorporated the changes in another branch that fully migrates to the Sphinx setup, which I will submit a PR for shortly.

**Comment by Vinnl** [comment]
New PR: #324.

**Pull request closed without merging**